less/index.less

```diff
@@ -81,10 +88,22 @@ header {
         justify-content: space-between;
         display: flex;
 
+        @media( max-width: 800px ) {
+        margin-top: 40px; 
```

Comment:
This media query for example is one area where you could use more indentation in your code, from _margin-top_ at line 92 through _display_ at line 95.